modules/AlphaWalletFoundation/AlphaWalletFoundation/Transfer/TransactionConfigurator.swift

```diff
@@ -243,37 +242,6 @@ public class TransactionConfigurator {
                 try encoder.encode(function: functionEncoder, arguments: parameters)
                 let gasLimit = transaction.gasLimit ?? maxGasLimit
 
-                return createConfiguration(server: server, analytics: analytics, transaction: transaction, gasLimit: gasLimit, data: encoder.data)
-            case .erc875TokenOrder(let token, _):
-                guard let expiry = transaction.expiry, let indices = transaction.indices, let v = transaction.v, let r = transaction.r, let s = transaction.s else {
-                    throw TransactionConfiguratorError.impossibleToBuildConfiguration
-                }
-                let parameters: [Any] = [
-                    expiry,
-                    indices.map({ BigUInt($0) }),
-                    BigUInt(v),
-                    Data(_hex: r),
-                    Data(_hex: s)
-                ]
-
-                let arrayType: ABIType
-                if token.contractAddress.isLegacy875Contract {
-                    arrayType = ABIType.uint(bits: 16)
-                } else {
-                    arrayType = ABIType.uint(bits: 256)
-                }
-
-                let functionEncoder = Function(name: "trade", parameters: [
```

Comment:
Is this functionality moved somewhere? Or is it no longer used because the `.erc875TokenOrder` enum-case is never referred to?